### ISSUE TITLE
training alignment

### ISSUE THREAD

**Issue by PanXiebit** [baaivision/MUSE-Pytorch#5]
Thanks for your great works! I am trying to train the model on laion2B datasets.  

My training details are:  base_size=4096, learning_rate=1e-4, AdamW optimizer.

After 130000 iterations, the loss curve is as follows, it seems to flatten...
![image](https://github.com/baaivision/MUSE-Pytorch/assets/24931560/cbb95f75-26f0-456e-8f5c-bf3498f5fd87)

And the visualized images are:

![image](https://github.com/baaivision/MUSE-Pytorch/assets/24931560/96972f75-855d-470c-aa32-56d6fbd4ecaa)
prompts are:
A tiger in a lab coat with a 1980s Miami vibe, turning a well oiled science content machine, digital art
A group photo
a fish on a bike
star citizen aurora

Are you in a similar situation during training, what is the lowest loss you can achieve during training?? Hope to get your suggestions!

**Comment by liqiangniu** [comment]
How many gpu are you used?
A100 or V100?

**Comment by liqiangniu** [comment]
@PanXiebit
